src/test/java/com/ibm/cloudant/kafka/schema/ConnectRecordMapperTests.java

```diff
@@ -45,6 +59,22 @@ public void testConvertToMapNoSchema() {
         assertEquals("world", converted.get("hello"));
     }
 
+    @Test
```

Comment:
What do you think about having a negative test, eg where the header value is not a string, to show that conversion doesn't blow up?

Reply:
I've added two tests in 097f306:
- Convert to map with no schema, existing `_id` field, and invalid map header.  Assert that the `_id` field never changed.
- Convert to struct with invalid boolean header.  Assert that the `_id` field is `null`.